src/appleseed/renderer/modeling/light/maxomnilight.cpp

```diff
@@ -115,6 +115,23 @@ namespace
         virtual void sample(
             const ShadingContext&   shading_context,
             const Transformd&       light_transform,
+            const Vector3d&         target_point,
+            const Vector2d&         s,
+            Vector3d&               position,
+            Vector3d&               outgoing,
+            Spectrum&               value,
+            float&                  probability) const override
+        {
+            position = light_transform.get_parent_origin();
+            outgoing = normalize(target_point - position);
+            value = m_values.m_intensity;
+            probability = RcpFourPi<float>();
```

Comment:
Since there is no sampling, I believe that in this case `probability` should be 1.